docs/source/mr/index.rst

```diff
@@ -16,7 +16,7 @@ Examples
 
 ::
 
-  $ glab mr create --fill --labels bugfix
+  $ glab mr create --fill --label bugfix
```

Comment:
You don't need to update this. It's automatically generated. The change in the Example is enough. So if u can remove this change this is good to go